templates/partials/cameraControls.mustache

```diff
@@ -0,0 +1,15 @@
+<div id="cameraControls">
+	<div class="header">
+		<i class="icon-videocam"></i>Camera style:
+	</div>
+	<div class="options">
+		<div id="cameraFree" data-type="FreeCamera">
```

Comment:
I don't know if it's *that* worth changing, but maybe we shouldn't use `cameraFree` and `freeCamera` interchangeably. It's a bit confusing.

Reply:
I see your point, but in this case I wanted to stay consistent with the rest of the DOM. This section is all about cameras, so I figured I would prefix all IDs with `camera`
Maybe `cameraOptionFree` and `cameraOptionTracking` as a less confusing alternative?

Reply:
I see your point. Since we'll switch to proper constants, the confusion won't be there as much. Maybe just leave it as is in this case?